apps/datahub/src/main.ts

```diff
@@ -1,6 +1,9 @@
-import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { AppModule } from './app/app.module';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic'
+import { AppModule } from './app/app.module'
+import { loadAppConfig } from 'geonetwork-ui'
```

Comment:
Nice idea to reuse the existing `app-config` functions from gn-ui. I'm wondering if we should go this way, though, or rather implement only the functions we need here in MEL. Even though this will mean some identical code, it will allow removing the currently mandatory parameters that do not have any effect in the app and make for a cleaner config file. WDYT?

Reply:
Yes, I considered it, but indeed it's a lot more code than just "loadAppConfig", as it calls many other functions and models. I thought it was maybe a bit too much compared to 5 or 6 unused params in the config file. Also, the theme is handled quite differently in MEL so I thought it would not clash... But I'm happy to make the change if you think it best :wink: 

I'm also not sure to understand what change you mean in the Dockerfile?

Reply:
So as discussed, I'd go for a separate implementation of the `app-config`.